en/sql/query/json_table.rst

```diff
@@ -0,0 +1,151 @@
+:meta-keywords: cubrid json, cubrid json_table, database json
+:meta-description: CUBRID JSON_TABLE description
+
+*********************************
+JSON_TABLE
+*********************************
+
+**JSON_TABLE** facilitates transforming of json data into a table having the specified columns.
+
+The full syntax of **JSON_TABLE**:
+::
+    JSON_TABLE(
+        expr,
+        path COLUMNS (column_list)
+    )   [AS] alias
+
+
+    <column_list>::=
+        <column> [, <column>] ...
+
+    <column>::=
+        name FOR ORDINALITY
+	|  name type PATH string path <on_empty> <on_error>
+	|  name type EXISTS PATH string path
+	|  NESTED [PATH] path COLUMNS <column_list>
+
+    <on_empty>::=
+        NULL | ERROR | DEFAULT json_value ON EMPTY
+
+    <on_error>::=
+        NULL | ERROR | DEFAULT json_value ON ERROR
+
+
+The json_doc expr must be an expression that results in a json_doc. This can be a constant json, a table's column or a function or operator result.
+The json path must be a valid path and is used to extract json data to be evaluated in the COLUMNS clause.
+The COLUMNS clause defines output column types and operations performed to get the output.  
+The [AS] alias clause is required.
+
+
+**JSON_TABLE** supports four types of columns:
+
+- name FOR ORDINALITY: this type keeps track of a row's number inside a COLUMNS clause. The column's type is INTEGER.
+- name type PATH json path [on empty] [on error]: Columns of this type are used to extract json_values from the specified json paths. The extracted json data is then coerced to the specified type.
+  If the path does not exist, json value triggers the on empty clause. The on error clause is triggered if the extracted json value is a json array, a json object or if the extracted json value is not coercible to the target type.
+
+  - on empty determines the behavior of JSON_TABLE in case the path does not exist. On empty can have one of the following values:
+
+    - NULL ON EMPTY: the column is set to NULL. This is the default behavior.
+    - ERROR ON EMPTY: an error is thrown
+    - DEFAULT json_string ON EMPTY: json_string will be used instead of the missing value. Column type rules also apply to the default value.
+
+  - on error can have one of the following values:
+
+    - NULL ON ERROR: the column is set to NULL. This is the default behavior.
+    - ERROR ON ERROR: an error is thrown.
+    - DEFAULT json_string ON ERROR: json_string will be used instead of the array/object/ json scalar that failed coercion to desired column type. Column type rules also apply to the default value. 
```

Comment:
```suggestion
    - DEFAULT json_string ON ERROR: json_string will be used instead of the array/object/json scalar that failed coercion to desired column type. Column type rules also apply to the default value. 
```